abci/cmd/abci-cli/abci-cli.go

```diff
@@ -600,6 +620,66 @@ func cmdQuery(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func inTxArray(txByteArray [][]byte, tx []byte) bool {
+	for _, txTmp := range txByteArray {
+		if bytes.Equal(txTmp, tx) {
+			return true
+		}
+
+	}
+	return false
+}
+func cmdPrepareProposal(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		printResponse(cmd, args, response{
+			Code: codeBad,
+			Info: "Must provide at least one transaction",
+			Log:  "Must provide at least one transaction",
+		})
+		return nil
+	}
+	txsBytesArray := make([][]byte, len(args))
+
+	for i, arg := range args {
+		txBytes, err := stringOrHexToBytes(arg)
+		if err != nil {
+
+			return err
+		}
+		txsBytesArray[i] = txBytes
+	}
+
+	res, err := client.PrepareProposal(cmd.Context(), &types.RequestPrepareProposal{
+		Txs: txsBytesArray,
+		// kvstore has to have this parameter in order not to reject a tx as the default value if 0
+		MaxTxBytes: 120,
```

Comment:
You might even want to put a way bigger value here, to avoid spurious errors if someone tests with a bunch of transactions. Maybe something not likely to be hit via cli (I don't know... 65536)?